src/config.js

```diff
@@ -954,7 +954,64 @@ let config = {
         q14: {answer: null},
       },
       addedBy: 'Mostafa Balata',
-    }
+    },
+    '0x71c2683152b632Fa1D515a3Df067825e2b240216': {
+      tokenContract: '0x0e9C4d8610Aa2cf0DaBDAbB19920A1C810B65720',
+      information: {
+        aliasName: 'KICKICO',
+        logo: 'https://www.kickico.com/favicon.png',
+        website: 'https://www.kickico.com/',
+      },
+      events: {
+        FundTransfer: {
+          args: {
+            tokens: null, // tokens not generated here, just ether gathered
```

Comment:
According to the Smart contract the tokens value is not null, it's `amount`.
`event FundTransfer(address backer, uint amount, bool isContribution);`